src/client/components/vision2/camera/model.ts

```diff
@@ -0,0 +1,52 @@
+import { observable } from 'mobx'
+
+import { Matrix4 } from '../../../math/matrix4'
+import { Vector2 } from '../../../math/vector2'
+import { Vector3 } from '../../../math/vector3'
+import { Image } from '../image'
+
+export class CameraImage {
+  @observable.ref image: Image
+  @observable.ref Hcw: Matrix4
+  @observable.ref lens: Lens
+
+  constructor(
+    image: Image,
+    Hcw: Matrix4,
+    lens: Lens,
+  ) {
+    this.image = image
+    this.Hcw = Hcw
+    this.lens = lens
+  }
+}
+
+export interface Lens {
+  projection: Projection,
+  normalizedFocalLength: number,
```

Comment:
I think it's still just focalLength. Normalised just describes what units it's in. And since it's normalised all through this codebase adding that is a little redundant.